### PR TITLE
snes.xml: Added nine dumps.

### PR DESCRIPTION
**Pull request by 0kmg**
New working software list additions
-----------------------------------
Doraemon 2 - Nobita no Toys Land Daibouken (Japan, prototype) [atomic]
Gourmet Warriors [NoIntro]
Jelly Boy (prototype) [Hard4Games, snescentral]
Jelly Boy (earlier prototype) [Mark Flitman, snescentral, VGHF]
Jimmy Connors Pro Tennis Tour (Germany, prototype) [snescentral]
Pinkie [NoIntro]
Superman Unfinished Version (demo) [Nostalgia Alley, snescentral, VGHF]
Tinhead [NoIntro]

New NOT_WORKING software list additions
---------------------------------------
Total Carnage (USA, prototype 19930610) [snescentral]

### CODE REVIEW

File: hash/snes.xml [hunk 1]
```diff
@@ -3541,6 +3566,30 @@ Beyond that last category are the roms waiting to be classified.
 		</part>
 	</software>
 
+	<software name="superman">
+		<!-- snescentral.com -->
+		<description>Superman Unfinished Version (demo)</description>
```
Comment (einstein95) [inline hash/snes.xml:3571]: Judging by the title screen from https://youtube.com/watch?v=jRsEBvgA33E, "Unfinished Version" is a subtitle, so it should read "Superman - Unfinished Version"

**Comment by 0kmg** [comment]
@Tafoid Since you're privately tracking SNES regressions, I thought I might mention Total Carnage. It seems none of the clones respond to inputs except maybe Start on the title screen. I can't confirm this is a regression since it's not easy for me to build 2-3 year old MAME trees on my Mac to compare with.

**Comment by cuavas** [comment]
> I can't confirm this is a regression since it's not easy for me to build 2-3 year old MAME trees on my Mac to compare with.

Can you use the builds @mac-a-r0ni publishes to check things?  https://sdlmame.lngn.net/stable/

**Comment by Tafoid** [comment]
> @Tafoid Since you're privately tracking SNES regressions, I thought I might mention Total Carnage. It seems none of the clones respond to inputs except maybe Start on the title screen. I can't confirm this is a regression since it's not easy for me to build 2-3 year old MAME trees on my Mac to compare with.

I'll jot it down and look into it.  Thanks.

**Comment by 0kmg** [comment]
@Tafoid I tested them in 0.208 (thanks to Vas' link and mac-a-r0ni's builds). They seemed to be working then. In current (and recent) MAME I can't move or shoot in the parent or any clone.

**Pull request merged**